packages/zipkin/src/instrumentation/httpServer.js

```diff
@@ -51,14 +51,14 @@ class HttpServerInstrumentation {
         });
       });
     } else {
-      if (readHeader(Header.Flags) !== None) {
+      if (readHeader(Header.Flags) !== None || readHeader(Header.Sampled) !== None) {
```

Comment:
I think this change needs a test case... looks like the intent is to also allow 'X-B3-Sampled' without trace identifiers... correct? similar to https://github.com/openzipkin/brave/blob/master/brave/src/main/java/brave/propagation/B3Propagation.java#L125

Reply:
I added tests for both: https://github.com/openzipkin/zipkin-js/pull/222/files#diff-69d935ef65c95d1891e05297e13f46aaR118